modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -165,7 +161,7 @@ void AudioStreamMP3::set_data(const Vector<uint8_t> &p_data) {
 
 	mp3dec_ex_t mp3d;
 	int err = mp3dec_ex_open_buf(&mp3d, src_datar, src_data_len, MP3D_SEEK_TO_SAMPLE);
-	ERR_FAIL_COND(err != 0);
+	ERR_FAIL_COND(err || mp3d.info.hz == 0);
```

Comment:
Maybe this can be given an explicit error message with the `_MSG` variant of the macro, as otherwise this might be a bit cryptic to users (especially as this often would happen in a chain reaction of failed loading errors).

Reply:
Good point! I've gone ahead and updated the commit; should hopefully be all good now.